support/doc/production.md

```diff
@@ -221,7 +221,7 @@ Make a SQL backup
 ```
 $ SQL_BACKUP_PATH="backup/sql-peertube_prod-$(date -Im).bak" && \
     cd /var/www/peertube && sudo -u peertube mkdir -p backup && \
-    sudo pg_dump -U peertube -W -h localhost -F c peertube_prod -f "$SQL_BACKUP_PATH"
+    sudo -u postgres pg_dump -F c peertube_prod | sudo -u peertube tee "$SQL_BACKUP_PATH" >/dev/null
```

Comment:
Why do you prefer to use `tee` command instead of `-f`?

Reply:
Since the `pg_dump` command is run as the `postgres` the file can't be written where `postgres` doesn't have permissions. The only option is to pipe the dump stdout to `tee` (either just with `sudo` - `root` will be owner, or as `peertube`)

Reply:
:+1: